math/src/field/traits.rs

```diff
@@ -258,11 +273,21 @@ pub trait StarkField: FieldElement<BaseField = Self> {
         Self::TWO_ADIC_ROOT_OF_UNITY.exp(power)
     }
 
-    /// Returns byte representation of the field modulus in little-endian byte order.
-    fn get_modulus_le_bytes() -> Vec<u8>;
+    /// Converts a slice of bytes into a field element. Pads the slice if it is smaller than the number
+    /// of bytes needed to represent an element.
+    ///
+    /// # Panics
+    /// Panics if the length of `bytes` is smaller than the number of bytes needed to encode an element.
```

Comment:
We panic if the number of bytes is greater than (not smaller than) the number of bytes for the field elements, right?

Also, another possible panic condition is when the number of bytes is correct but the value contained in these bytes is not a valid field element, right?

Reply:
Oops; right.

Fixed